samples/tools/finetuning/tests/test_conversable_agent_update_model.py

```diff
@@ -0,0 +1,222 @@
+import pytest
+from autogen import AssistantAgent, UserProxyAgent
+import sys
+import os
+
+sys.path.append("samples/tools/finetuning")
+
+from finetuning import update_model  # noqa: E402
+from typing import Dict  # noqa: E402
+
+sys.path.append("test")
+from conftest import skip_openai  # noqa: E402
+
+TEST_CUSTOM_RESPONSE = "This is a custom response."
+TEST_LOCAL_MODEL_NAME = "local_model_name"
+
+
+@pytest.mark.skipif(skip_openai, reason="requested to skip openai tests")
```

Comment:
Actually all the tests here do not require OpenAI. So we don't need to skip.

Reply:
they don't explicitly require OpenAI but they use the OAI wrapper and that will fail if OpenAI is not available